hbase-client/src/main/java/org/apache/hadoop/hbase/client/Admin.java

```diff
@@ -2254,4 +2256,70 @@ boolean snapshotCleanupSwitch(final boolean on, final boolean synchronous)
    */
   boolean isSnapshotCleanupEnabled() throws IOException;
 
+  /**
+   * Gets group info for the given group name
+   * @param groupName the group name
+   * @return group info
+   * @throws IOException if a remote or network exception occurs
+   */
+  RSGroupInfo getRSGroupInfo(String groupName) throws IOException;
+
+  /**
+   * Move given set of servers to the specified target RegionServer group
+   * @param servers set of servers to move
+   * @param targetGroup the group to move servers to
+   * @throws IOException if a remote or network exception occurs
+   */
+  void moveServers(Set<Address> servers, String targetGroup) throws IOException;
```

Comment:
Should all RS Grouping commands have a common prefix? You have RSGroup on some of them but not all such as this one.

Reply:
Yes, I'll do that.

Reply:
Yes, I'll do that.